packages/server/index.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const mongoose = require('mongoose');
+const app = express();
+require('dotenv').config();
+
+const { MONGODB_URI = 'mongodb://localhost:27017/tictac' } = process.env;
```

Comment:
Please destructure both MONGODB_URI and PORT from same line

```suggestion
const { MONGODB_URI = 'mongodb://localhost:27017/tictac', PORT = 3000 } = process.env;
```